packages/amplify-category-function/src/commands/function/update.ts

```diff
@@ -19,7 +22,15 @@ module.exports = {
         }
         return providerController.updateResource(context, categoryName, result.service);
       })
-      .then(() => context.print.success('Successfully updated resource'))
+      .then(result => {
+        const { print } = context;
+        const { name, service } = result;
+        if (service === ServiceName.LambdaFunction) {
```

Comment:
This if/else statement is currently doing the same thing for both services

Reply:
fixed